valueformatting_test.go

```diff
@@ -267,6 +267,30 @@ func TestValueFormattingBinaryFormatter(t *testing.T) {
 	}
 }
 
+func TestValueFormattingJSONFormatter(t *testing.T) {
+	formatting := newValueFormatting()
```

Comment:
A shorter variable here is ok since its single use is immediately below. Maybe `vf`?

Reply:
Done.